portal/templates/redesign/teach_new/dashboard.html

```diff
@@ -79,53 +72,66 @@ <h4 class="col-sm-6 col-center">As an administrator of your school or club, you
             <tr>
                 <th><p class="body-text">Name</p></th>
                 <th><p class="body-text">Administrator status</p></th>
-                <th colspan="3"><p class="body-text">Status actions</p></th>
-            </tr>
-            <tr>
-                <td><p class="small-text">Mr James Holland (you)</p></td>
-                <td><p class="small-text">Administrator (**********@gmail.com)</p></td>
-                <td class="cell-center"><button class="button--small button-primary button--primary--danger">Remove</button></td>
-                <td class="cell-center"><button class="button--small button-primary button--primary--navigation">Make admin</button></td>
-                <td class="cell-center"><button class="button--small button-primary button--primary--setup">Disable 2FA</button></td>
-            </tr>
-            <tr>
-                <td><p class="small-text">Ms Naomie Noobie</p></td>
-                <td><p class="small-text">Standard teacher (**********@gmail.com)</p></td>
-                <td class="cell-center"><button class="button--small button-primary button--primary--danger">Remove</button></td>
-                <td class="cell-center"><button class="button--small button-primary button--primary--navigation">Make admin</button></td>
-                <td class="cell-center"><button class="button--small button-primary button--primary--setup">Disable 2FA</button></td>
+                {% if is_admin %}
+                    <th colspan="3"><p class="body-text">Status actions</p></th>
+                {% endif %}
             </tr>
+            {% for coworker in coworkers %}
             <tr>
-                <td><p class="small-text">Mrs Omya Overloaded</p></td>
-                <td><p class="small-text">Standard teacher (**********@gmail.com)</p></td>
-                <td class="cell-center"><button class="button--small button-primary button--primary--danger">Remove</button></td>
-                <td class="cell-center"><button class="button--small button-primary button--primary--navigation">Make admin</button></td>
-                <td></td>
+                <td><p class="small-text">{{ coworker.title }} {{ coworker.new_user.first_name }} {{ coworker.new_user.last_name }}
+                    {% if coworker.new_user == user %}<b>(you)</b>{% endif %}</p></td>
```

Comment:
The (you)  will need to be externalised at some point - to remember :) Especially it might not be on the right of the name, but on the left for right to left languages.